### PR TITLE
Performance improvements

### PR DESCRIPTION
**Pull request by cquiroz**
And a few other small fixes

### CODE REVIEW

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala [hunk 1]
```diff
@@ -129,7 +126,7 @@ trait VoTableParser {
         .andThen(parseDoubleDegrees(VoTableParser.UCD_RA, _))
         .map(a => RightAscension.fromDoubleDegrees(a.toDoubleDegrees))
 
-    val parsePV = adapter.parseProperMotion(entries)
+    def parsePV = adapter.parseProperMotion(entries)
```
Comment (rpiaggio) [inline modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:129]: Why did this change from `val` to `def` ?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala (hunk 1 above)
Comment (cquiroz) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:129]: we don't want to eagerly load a value which may not be needed if other parts fail to validate

Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:129]: `lazy val` ?

**Pull request merged**